vdj_ann/src/vdj_features.rs

```diff
@@ -416,11 +416,11 @@ pub fn fr3_start(aa: &[u8], chain_type: &str, verbose: bool) -> Option<usize> {
 
         // Score positions.
 
-        if cdr3_start < 35 {
+        if cdr3_start < 28 {
             return None;
         }
         let mut score_pos = Vec::<(usize, usize)>::with_capacity(35 - 28 + 1);
-        for j in cdr3_start - 35..=cdr3_start - 28 {
+        for j in cdr3_start.saturating_sub(35)..=cdr3_start - 28 {
```

Comment:
It's weird that the `.saturating_sub` pattern seems to have been used all the cases after this but not here ?

Reply:
I suspect that the `IGH` case failed during development and it was fixed by applying saturation to both ends of the range, which is either a bug or a very perplexing and uncommented special case.